### DISCUSSION TITLE
Coordinating major themes collection for a release

### DISCUSSION THREAD

**Discussion by wilsonehusin** [kubernetes/sig-release#1392] [General]
As part of a Kubernetes release, the release team compiles a list of _major themes_. We have multiple publish locations for this and they have varying content. For example: [release blog](https://kubernetes.io/blog/2020/12/08/kubernetes-1-20-release-announcement/#major-themes) and [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md#whats-new-major-themes).

In Kubernetes 1.20, we have coordinated effort of a primary file on compiling the entries to this list. This discussion is opened to find out how we can further formalize the process.

Some goals I can think about (please weigh in / add / remove):
- **Consistency** between published medias -- or not! I think we need justifications for either
- **Structured workflow** such that SIG Chairs / PR authors will reach out to us (release team) if they have entries to submit, such that:
  - SIG Chairs _know_ that they need to submit it to release team
  - Release team _know_ if something needs to be added (e.g. on PR for feature gate change, ask if this should go to major themes as well)
  - This will also [pave the way for automation](https://github.com/kubernetes/release/pull/1820), similar to how [each PR asks for release notes](https://github.com/kubernetes/kubernetes/blob/master/.github/PULL_REQUEST_TEMPLATE.md)
- **Early start** on compilation so that we leave enough time for reviewers (i.e. technical writers / SIG Docs), instead of rushing everything on the last week or two

With all that being said, I'd like to invite the 1.21 enhancement, docs, & comms leads to start! (of course, everyone is welcomed :smile:)

cc: @annajung @reylejano @divya-mohan0209

**Comment by justaugustus** [comment]
Please add this as part of https://github.com/kubernetes/community/issues/5344.

/close
